developer/variables.pkrvars.hcl

```diff
@@ -2,4 +2,5 @@ SUBNET_ID = " "
 REGION = " "
 SECURITY_GROUP_ID = " "
 RESOURCE_GROUP_ID = " "
-IBM_API_KEY = " "
\ No newline at end of file
+IBM_API_KEY = " "
+ENCRYPTION_KEY_CRN = ""
```

Comment:
` ENCRYPTION_KEY_CRN = ""` has to be `ENCRYPTION_KEY_CRN = " "`
`""` messes up with some build I am hard time trying to recall

Reply:
addressed.